src/xcode/ENA/ENA/Source/Services/CCLService/DCCUIText.swift

```diff
@@ -0,0 +1,273 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import AudioToolbox
+import AnyCodable
+import jsonlogic
+
+enum ParameterType {
+	static let number = "number"
+	static let string = "string"
+	static let localDate = "localDate"
+	static let localDateTime = "localDateTime"
+	static let utcDate = "utcDate"
+	static let utcDateTime = "utcDateTime"
+}
+
+enum UITextType {
+	static let string = "string"
+	static let plural = "plural"
+	static let systemTimeDependent = "system-time-dependent"
+}
+
+struct DCCUITextParameter {
+	let type: String
+	let value: Any
+}
+
+public struct DCCUIText: Codable, Equatable {
+	let type: String
+	let quantity: Int?
+	let quantityParameterIndex: Int?
+	let functionName: String?
+	let localizedText: [String: AnyCodable]?
+	let parameters: AnyCodable
+
+	static let dateFormatter: ISO8601DateFormatter = .iso8601DateFormatter()
```

Comment:
I guess you are using these date formatters for similar strings like I was using in the other task. We should consider to merge them.

Reply:
Output of all four formatters are different as that of your formatters, so I would like to stick with the ones used here. 

Unit tests:

XCTAssertEqual failed: ("Optional("Gültig ab 2022-01-01")") is not equal to ("Optional("Gültig ab 01.01.22")") - Failing Test: placeholder of type utcDate

XCTAssertEqual failed: ("Optional("Gültig ab 2022-01-01T23:30:00Z")") is not equal to ("Optional("Gültig ab 01.01.22, 23:30")") - Failing Test: placeholder of type utcDateTime

XCTAssertEqual failed: ("Optional("Gültig ab 2022-01-02")") is not equal to ("Optional("Gültig ab 02.01.22")") - Failing Test: placeholder of type localDate

XCTAssertEqual failed: ("Optional("Gültig ab 2022-01-02T00:30:00+01:00")") is not equal to ("Optional("Gültig ab 02.01.22, 00:30")") - Failing Test: placeholder of type localDateTime